src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1370,7 +1377,7 @@ new PrismaClient({
             } else {
               acc[key] = value
             }
-            if (key === 'count') {
+            if (key === '_count') {
```

Comment:
Should we support `count` here too?

Reply:
No, as `count` is no longer valid on groupBy. This will be the breaking change